core/types/src/measurement.rs

```diff
@@ -24,14 +25,29 @@ pub struct MrEnclave(sgx_measurement_t);
 #[repr(transparent)]
 pub struct MrSigner(sgx_measurement_t);
 
-impl_newtype_for_bytestruct! {
+impl_newtype_for_bytestruct_no_display! {
     MrEnclave, sgx_measurement_t, SGX_HASH_SIZE, m;
     MrSigner, sgx_measurement_t, SGX_HASH_SIZE, m;
 }
 
+impl Display for MrEnclave {
```

Comment:
IFF #310 moves the display logic into `impl_newtype_for_bytestruct` then these `Display` impls should go away too.

Reply:
Done.